fastlane/Fastfile

```diff
@@ -64,26 +72,55 @@ platform :ios do
     clean_build_artifacts
 
   end
+  
+  desc "New release to testflight for external users."
+  desc "Parameters:"
+  desc "- bump_type (optional): represents the type of deploy. If not specified, the user will be asked for it." 
+  lane :release_testflight do |options|
+    release build_configuration: :testflight, bump_type: options[:bump_type]
+  end
 
-  desc "Releases a new version to `TestFlight`. This lane uses the configuration mapped to `:testflight`."
+  desc "New release to testflight for QA. Must be build."
```

Comment:
I'd pick one out of these: `Must be built/Has to be built` or `Needs buliding` or `Needs Build`

Reply:
It should be `build`. It's referencing a `bump_type`, not being used as a verb. @damfinkel check the `bump_type`s are used between these marks to avoid these kind of misunderstadings.

Reply:
@dtilve @pablog added a commit updating the description. There's no need to specify anything regarding bump_types to the user for release_qa. It will always automatically use "build", and no bump_type parameter is allowed.